dataclients/kubernetes/kube.go

```diff
@@ -348,9 +349,19 @@ func shuntRoute(r *eskip.Route) {
 }
 
 func healthcheckRoute(healthy, reverseSourcePredicate bool) *eskip.Route {
-	status := http.StatusOK
+	logFilters := []*eskip.Filter{{
+		Name: builtin.StatusName,
+		Args: []interface{}{http.StatusOK}},
+	}
 	if !healthy {
-		status = http.StatusServiceUnavailable
+		logFilters[0].Args = []interface{}{http.StatusServiceUnavailable}
+	}
+	// log if unhealthy or a debug loglevel
+	if healthy && !log.IsLevelEnabled(log.DebugLevel) {
+		logFilters = append(logFilters, &eskip.Filter{
+			Name: accesslog.DisableAccessLogName,
+			Args: []interface{}{200},
+		})
```

Comment:
An alternative could be
```go
var healthcheckFilters []*eskip.Filter
	if healthy {
		if log.IsLevelEnabled(log.DebugLevel) {
			healthcheckFilters, _ = eskip.ParseFilters(`status(200)`)
		} else {
			healthcheckFilters, _ = eskip.ParseFilters(`disableAccessLog(200) -> status(200)`)
		}
	} else {
		healthcheckFilters, _ = eskip.ParseFilters(`status(503)`)
	}
```
Also related to #1537

Reply:
This passing is awesome to know!

My personal preference is to use static type compiler support whenever possible, not sure how expensive runtime parsing is but I’d expect a magnitude!?

Reply:
It is more about readabilty, skipper parses hundreds of routes on route table update anyway.

Reply:
We normally don’t parse filters like this in the dataclient if it’s not eskip or inline route